Gemfile.lock

```diff
@@ -24,7 +24,7 @@ DEPENDENCIES
   slack-notifier
 
 RUBY VERSION
-   ruby 2.7.0p0
+   ruby 3.1.0
 
 BUNDLED WITH
-   2.1.4
+   2.4.10
```

Comment:
Why is this reference still Ruby 2.4?

Reply:
The Latest Bundler is 2.4.10
 https://github.com/cloudfoundry/ruby-buildpack/blob/master/manifest.yml